learn-ocaml.opam.locked

```diff
@@ -74,6 +74,7 @@ depends: [
   "mmap" {= "1.1.0"}
   "num" {= "1.4"}
   "ocaml" {= "4.12.0"}
+  "ocaml-base-compiler" {= "4.12.0"}
```

Comment:
I think this line should be removed (the line `"ocaml" {= "4.12.0"}` is enough).

Otherwise learn-ocaml would be incompatible with an `ocaml-system` setup…